src/main/frontend/commands.cljs

```diff
@@ -554,7 +554,7 @@
                     (count (util/safe-re-find re-pattern prefix))))
             new-value (str (subs edit-content 0 pos)
                            (string/replace-first (subs edit-content pos)
-                                                 marker/marker-pattern
+                                                 (marker/marker-pattern)
```

Comment:
Should pass format from here instead of accessing state?

Reply:
block `format` is safe here, so I can pass it in the function. Thanks.